menuxconfig.py

```diff
@@ -0,0 +1,765 @@
+
+        self.helpmenu = tk.Menu(self.menu, tearoff=0)
+        self.helpmenu.add_command(
+            label="Current Symbol Help",
+            command=self.onHelp,
+            underline=15,
+            accelerator=' ctrl+h')
+        self.helpmenu.add_command(
+            label="Find Symbol",
+            command=self.onFind,
+            underline=0,
+            accelerator=' ctrl+f')
+        self.menu.add_cascade(
+            label="Symbol Information",
+            underline=0,
+            menu=self.helpmenu)
+        self.menu.add_command(
+            label="About",
+            command=self.onAbout,
+            underline=0)
+
+        self.root.bind_all('<Control-Key-s>', self.onSave)
+        self.root.bind_all('<Control-Key-q>', self.onQuit)
+        self.root.bind_all('<Control-Key-h>', self.onHelp)
+        self.root.bind_all('<Control-Key-f>', self.onFind)
+        self.root.protocol("WM_DELETE_WINDOW", self.onQuit)
+
+    def onAbout(self, ev=None):
+        description = {'About this project': information}
+        self.w = HelpPopupWindow(self.root, "About", description)
+        self.root.wait_window(self.w.top)
+
+    def onSave(self, ev=None):
+        try:
+            if self.header is None:
+                self.conf.write_config(self.cfile)
+            else:
+                self.conf.write_config(self.cfile, header=self.header)
+        except Exception as exc:
+            tkmsgbx.showerror(
+                "Fail save",
+                "Failed to Save configuration to" +
+                self.cfile +
+                "!" + str(exc))
+            raise(exc)
+            return
+
+        tkmsgbx.showinfo(
+            "Saved config",
+            "Configurations Saved to " +
+            self.cfile +
+            "!")
+        self.modified = False
+
+    def onSaveAs(self):
+        filename = tkfile.asksaveasfilename(
+            title="Select Config file to save to")
+        if not filename:
+            return
+
+        self.cfile = filename
+        self.onSave()
+
+    def onQuit(self, ev=None):
+        if self.modified:
+            if tkmsgbx.askyesno(
+                    "Save Configuration?", "Configuration has been modified: Save this config?"):
+                self.onSave()
+        self.root.quit()
+
+    def onHelp(self, ev=None, sym=None):
+        if sym is None:
+            sym = self.symbol
+        if sym is None:
+            return
+
+        help_txt = sym.nodes[0].help
+        info_txt = sym.__str__()
+        if help_txt is None:
+            help_txt = "No Help defined"
+
+        information = {
+            "Help Text:": help_txt,
+            "Symbol Information:": info_txt
+        }
+        self.w = HelpPopupWindow(self.root, sym.name, information)
+        self.root.wait_window(self.w.top)
+
+    def onFind(self, ev=None):
+        self.pop = TextPopupWindow(self.root, "Enter Name of Symbol", "")
```

Comment:
Could this be a local variable instead? It looks like `self.pop` is always set at the start of a function, and then never used outside it.